packages/dids/src/did-dht.ts

```diff
@@ -0,0 +1,225 @@
+import z32 from 'z32';
+import crypto from 'crypto';
```

Comment:
`crypto` isn't available in the browser unless we polyfill it. AFAICT you're only using `crypto` to generate digests which we're using [`@noble/hashes`](https://www.npmjs.com/package/@noble/hashes) for. [examples here](https://github.com/search?q=repo%3ATBD54566975%2Fweb5-js+%40noble%2Fhashes%2Fsha&type=code)

@frankhinek, i forget, did we end up deciding it'd be worth adding common hashing functions to `common`?

Reply:
worked for me